.appveyor.yml

```diff
@@ -30,7 +30,7 @@ install:
     - echo extension=php_openssl.dll >> php.ini
     - echo extension=php_mbstring.dll >> php.ini
     - IF NOT EXIST C:\tools\composer.phar (cd C:\tools && appveyor DownloadFile https://getcomposer.org/composer-stable.phar)
-    - rename C:\tools\composer-stable.phar composer.phar
+    - rename composer-stable.phar composer.phar
```

Comment:
Should that second line be merged withing the first? i.e.:
```yml
     - IF NOT EXIST C:\tools\composer.phar (cd C:\tools && appveyor DownloadFile https://getcomposer.org/composer-stable.phar && rename composer-stable.phar composer.phar)
```

Reply:
That's the fix I intend to push, just checking :)

Reply:
Apparently you can even simplify
`appveyor DownloadFile https://getcomposer.org/composer-stable.phar && rename composer-stable.phar composer.phar` into
`appveyor DownloadFile https://getcomposer.org/composer-stable.phar -FileName composer.phar`

Reply:
Nice, thanks!